### PR TITLE
adds documentation for application delta v1.0

### PR DESCRIPTION
**Pull request by baywet**
@sureshja I'm trying to close the gaps we have in terms of delta query support across versions.
From the metadata and my tests it looks like delta for applications made it to v1.0 but this has never been documented. Can you review the content please?

### CODE REVIEW

File: api-reference/v1.0/api/application-delta.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+---
+title: "application: delta"
+description: "Get newly created, updated, or deleted applications without having to perform a full read of the entire resource collection. See Using Delta Query for details."
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:3]: We try to keep these as short as possible and still convey the message. Suggestion: Get created, updated, or deleted applications without having to perform a full read of the entire resource collection.

File: api-reference/v1.0/api/application-delta.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+|:---------------|:--------|:----------|
+| $deltatoken | string | A [state token](/graph/delta-query-overview) returned in the `deltaLink` URL of the previous **delta** function call for the same resource collection, indicating the completion of that round of change tracking. Save and apply the entire `deltaLink` URL including this token in the first request of the next round of change tracking for that collection.|
+| $skiptoken | string | A [state token](/graph/delta-query-overview) returned in the `nextLink` URL of the previous **delta** function call, indicating there are further changes to be tracked in the same resource collection. |
+
+## Optional query parameters
+
+This method supports OData Query Parameters to help customize the response.
+
+- You can use a `$select` query parameter as in any GET request to specify only the properties your need for best performance. The 
+_id_ property is always returned.
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:56]: Not sure why this is on a separate line.

File: api-reference/v1.0/api/application-delta.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+## Request headers
+| Name       | Description|
+|:---------------|:----------|
+| Authorization  | Bearer &lt;token&gt;|
+| Content-Type  | application/json |
+
+## Request body
+Do not supply a request body for this method.
+
+### Response
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:71]: This should be an H2: ##Response

File: api-reference/v1.0/api/application-delta.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+If successful, this method returns `200 OK` response code and [application](../resources/application.md) collection object in the response body. The response also includes a nextLink URL or a deltaLink URL.
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:73]: If successful, this method returns a 200 OK response code...

Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:73]: Be consistent with "nextLink URL or a deltaLink URL"

File: api-reference/v1.0/api/application-delta.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+- If a `nextLink` URL is returned, there are additional pages of data to be retrieved in the session. The application continues making requests using the `nextLink` URL until a `deltaLink` URL is included in the response.
+
+- If a `deltaLink` URL is returned, there is no more data about the existing state of the resource to be returned. Persist and use the `deltaLink` URL to learn about changes to the resource in the future.
+
+See:</br>
+- [Using Delta Query](/graph/delta-query-overview) for more details</br>
+- [Get incremental changes for users](/graph/delta-query-users) for an example requests.</br>
+
+### Example
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:83]: This should be an H2: ##Example

File: api-reference/v1.0/api/application-delta.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+If successful, this method returns `200 OK` response code and [application](../resources/application.md) collection object in the response body. The response also includes a nextLink URL or a deltaLink URL. 
+
+- If a `nextLink` URL is returned, there are additional pages of data to be retrieved in the session. The application continues making requests using the `nextLink` URL until a `deltaLink` URL is included in the response.
+
+- If a `deltaLink` URL is returned, there is no more data about the existing state of the resource to be returned. Persist and use the `deltaLink` URL to learn about changes to the resource in the future.
+
+See:</br>
+- [Using Delta Query](/graph/delta-query-overview) for more details</br>
+- [Get incremental changes for users](/graph/delta-query-users) for an example requests.</br>
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:81]: Get incremental changes for users for example requests.

File: api-reference/v1.0/api/application-delta.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+
+### Example
+##### Request
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:84]: This should be an H3: ###Request

File: api-reference/v1.0/api/application-delta.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "application_delta"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/applications/delta
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:94]: I think these code snippet entries should be removed. The correct snippets will then be automatically added.

File: api-reference/v1.0/api/application-delta.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+
+# [Objective-C](#tab/objc)
+[!INCLUDE [sample-code](../includes/snippets/objc/application-delta-objc-snippets.md)]
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+
+---
+
+
+##### Response
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:109]: This should be an H3: ###Response

File: api-reference/v1.0/api/application-delta.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+Note: The response object shown here may be truncated for brevity. All of the properties will be returned from an actual call.
```
Comment (davidmu1) [inline api-reference/v1.0/api/application-delta.md:110]: The format of the Note should be: >Note:

File: api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (davidmu1) [inline api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md:1]: These should not be in the PR. These snippets should not be manually edited. They are automatically generated once a week from the HTTP entry in the file.

File: api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+description: "Automatically generated file. DO NOT MODIFY"
```
Comment (davidmu1) [inline api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md:2]: See the comment above.

File: api-reference/v1.0/includes/snippets/javascript/application-delta-javascript-snippets.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (davidmu1) [inline api-reference/v1.0/includes/snippets/javascript/application-delta-javascript-snippets.md:1]: See the comment above.

File: api-reference/v1.0/includes/snippets/objc/application-delta-objc-snippets.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (davidmu1) [inline api-reference/v1.0/includes/snippets/objc/application-delta-objc-snippets.md:1]: See comment above.

File: api-reference/v1.0/api/application-delta.md (hunk 1 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:3]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 3 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:71]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 4 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:73]: thanks, fixed

Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:73]: I'm not sure I understand this comment, could you rephrase please?

File: api-reference/v1.0/api/application-delta.md (hunk 5 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:83]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 2 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:56]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 6 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:81]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 7 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:84]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 8 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:94]: I also copied the snippets we had in beta, should I still remove both the snippets and the references?

File: api-reference/v1.0/api/application-delta.md (hunk 9 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:109]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 10 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:110]: thanks, fixed

File: api-reference/v1.0/api/application-delta.md (hunk 8 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md:94]: nevermind, removed

File: api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md (hunk 11 above)
Comment (baywet) [reply api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md:1]: removed

File: api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md (hunk 12 above)
Comment (baywet) [reply api-reference/v1.0/includes/snippets/csharp/application-delta-csharp-snippets.md:2]: removed

File: api-reference/v1.0/includes/snippets/javascript/application-delta-javascript-snippets.md (hunk 13 above)
Comment (baywet) [reply api-reference/v1.0/includes/snippets/javascript/application-delta-javascript-snippets.md:1]: removed

File: api-reference/v1.0/includes/snippets/objc/application-delta-objc-snippets.md (hunk 14 above)
Comment (baywet) [reply api-reference/v1.0/includes/snippets/objc/application-delta-objc-snippets.md:1]: removed

File: api-reference/v1.0/api/application-delta.md (hunk 4 above)
Comment (baywet) [reply api-reference/v1.0/api/application-delta.md]: fixed
